### ISSUE TITLE
How to use i18n locale?

### ISSUE THREAD

**Issue by serek** [elevation/event_calendar#3]
Hello,

I would like to see month numbers in my language. I have a pl.yml (polish) in /config/locales/

I've also included in my locale file the standard date formats and more.

```
  date:
  formats:
  default: "%Y-%m-%d"
  short: "%d %b"
  long: "%d %B %Y"

  day_names: [Niedziela, Poniedziałek, Wtorek, Środa, Czwartek, Piątek, Sobota]
  abbr_day_names: [nie, pon, wto, śro, czw, pia, sob]

  month_names: [~, Styczeń, Luty, Marzec, Kwiecień, Maj, Czerwiec, Lipiec, Sierpień, Wrzesień, Październik, Listopad, Grudzień]
  abbr_month_names: [~, sty, lut, mar, kwi, maj, cze, lip, sie, wrz, paź, lis, gru]
  order: [ :year, :month, :day ]
```

**Comment by jeffschuil** [comment]
Closed by 40de9e1885e42e284b75ec4910db6e99de8f272c. Basic support for month and day name localization (il8n).

**Comment by serek** [comment]
Great!
